web/js/deerInitializer.js

```diff
@@ -220,15 +224,19 @@ DEER.TEMPLATES.Event = function (experienceData, options = {}) {
                 ${relatedSensesByName}
             </ul>
         `
+        if(date){
+            date = new Date(date).toLocaleString()
+        }
         let researchersHTML = `<dt>LRDA Researchers Involved</dt><dd><ul id="researchersInExperience">${contributorsByName}</ul></dd>`
         let peopleHTML = `<dt>People Involved</dt><dd><ul id="peopleInExperience">${peopleByName}</ul></dd>`
         let orgHTML = `<dt>Organizations Involved</dt><dd><ul id="organizationsInExperience">${organizationsByName}</ul></dd>`
         let placeHTML = `<dt>Location</dt><dd>${placeLabelHTML}</dd>`
-        let dateHTML = `<dt>Associated Date</dt><dd>${date}</dd>`
+        let dateHTML = `<dt>Associated Date and Time</dt><dd>${date}</dd>`
         let descriptionHTML = `<dt>Description</dt><dd>${description}</dd>`
+        let notesHTML = `<dt>General Notes</dt><dd>${notes}</dd>`
         let artifactsHTML = objectsHTML + practicesHTML + sensesHTML
 
-        tmpl += placeHTML + dateHTML + researchersHTML + peopleHTML + orgHTML + descriptionHTML + artifactsHTML + `</div>`
+        tmpl += placeHTML + dateHTML + researchersHTML + peopleHTML + orgHTML + descriptionHTML + notesHTML + artifactsHTML + `</div>`
```

Comment:
This is funny to have + concatenation with a backticked template string at the end, but you do you.